Gems/Camera/Code/Source/CameraComponentController.cpp

```diff
@@ -412,6 +415,66 @@ namespace Camera
         return m_isActiveView;
     }
 
+    namespace Util
+    {
+        AZ::Vector3 GetWorldPosition(const AZ::Vector3& origin, float depth, const AzFramework::CameraState& cameraState)
+        {
+            if (depth == 0.f)
+            {
+                return origin;
```

Comment:
suggestion: With these changes we've found it's actually preferable to return the position on the near clip plane as opposed to the actual origin point (as this can lead to weird picking behaviors where an object that is not visible can still be picked because it's between origin and the near plane).

Reply:
I think my variable name choice is to blame here.  In this case, the origin is on the near plane, or at least it appeared to be in my testing - definitely not the camera origin.

Reply:
Okay awesome good to know, thank you for clarifying 👍